channeld/channel_wire.csv

```diff
@@ -1,11 +1,11 @@
 # Received and sent funding_locked
-channel_normal_operation,1001
+channel_normal_operation,11001
```

Comment:
Why don't we add structure to the messages? We could enumerate the daemons and each daemon accepts `daemon_num | message_num` and fails anything else. That way we wouldn't need to manually make sure that these numbers are unique (and peer messages would simply get `daemon_num = 0`)

Reply:
That would have been more work, I just used sed on each daemon :)    But yeah, could just insert this in the generator, then assert that type & DAEMON_BITS is correct; I have another patch done which makes generator treat the bolt specs different from out CSV files, for 32-bit lengths, so it wouldn't be hard to add later.

I've added a FIXME :)